webui/react/src/pages/ExperimentDetails/ExperimentTrials.settings.ts

```diff
@@ -24,18 +24,18 @@ export const DEFAULT_COLUMNS: TrialColumnName[] = [
   'latestValidationMetric',
   'startTime',
   'duration',
-  'restarts',
+  'autoRestarts',
   'checkpoint',
 ];
 
 export const DEFAULT_COLUMN_WIDTHS: Record<TrialColumnName, number> = {
   action: 46,
+  autoRestarts: 117,
```

Comment:
Just a heads up - I think this portion seems to be ok and not impacted, but some id renames that cause local storage keys to no longer map to the new renamed key, can cause the UI to not load properly.

Reply:
I'll keep it in mind. Thanks for letting me know!